demo/Titanic/run_game.py

```diff
@@ -0,0 +1,123 @@
+from time import sleep
+
+from simulation.iceberg import Iceberg
+from simulation.journey import Journey
+from simulation.location_on_grid import LocationOnGrid, AreaOnGrid
+from simulation.map_animation import map_animation
+from simulation.ocean import Ocean
+import simulation.ocean
+from simulation.titanic_in_ocean import TitanicInOcean
+from system.titanic import Titanic
+
+locations = {
+    'Southampton': LocationOnGrid(50.909698, -1.404351),
+    'Cherbourg': LocationOnGrid(49.630001, -1.620000),
+    'Queenstown': LocationOnGrid(51.850334, -8.294286),
+    'New York': LocationOnGrid(40.730610, -73.935242)
+}
+
+areas = {
+    'The English Channel': AreaOnGrid(LocationOnGrid(49.5, -1.41), LocationOnGrid(51.9, -8.2)),
+    'Iceberg alley': AreaOnGrid(LocationOnGrid(43, -45), LocationOnGrid(48, -50))
+}
+
+atlantic_area = AreaOnGrid(LocationOnGrid(40, -1.41), LocationOnGrid(51.9, -74))
```

Comment:
The game area looked a bit weird (narrow with the legend hiding stuff) so I tried updating these values to 35, 65 which looked better overall. Only then the New York detection fails and you can't finish the game.